src/common/providers/identity.ts

```diff
@@ -835,9 +835,13 @@ export function wrapHandler(
         'skipTokenVerification'
       )
         ? unsafeDecodeAuthBlockingToken(req.body.data.jwt)
+        : handler.length === 2
+        ? await apps()
+            .admin.auth()
+            ._verifyAuthBlockingToken(req.body.data.jwt)
         : await apps()
             .admin.auth()
-            ._verifyAuthBlockingToken(req.body.data.jwt);
+            ._verifyAuthBlockingToken(req.body.data.jwt, 'run.app');
```

Comment:
q: I was checking to see when `_verifyAuthBlockingToken` function was introduced, and it seems like it's a pretty recent addition.

Because we support multiple version of the admin sdk (right now, all the way back to v8.0, though that will change in our next breaking release), I think we need to make sure that this code doesn't crash when using an older version.

e.g. we do something like that for app check here:

https://github.com/firebase/firebase-functions/blob/76e0afd442e141ad0016d01545e5883e27712c3f/src/common/providers/https.ts#L675-L678

Reply:
way ahead of me.